spec/tree-view-spec.coffee

```diff
@@ -663,7 +674,10 @@ describe "TreeView", ->
 
         children = root1.find('.directory')
         expect(children.length).toBeGreaterThan 0
-        children.each (index, child) -> expect(child).toHaveClass 'expanded'
+        children.each (index, child) ->
+          expect(child).toHaveClass 'expanded'
+          expect(child.hasAttribute('draggable')).toBe(true)
+          expect(child.getAttribute('draggable')).toBe('true')
```

Comment:
I could be wrong about this, but I interpret hasAttribute as being true regardless of whether draggable's value is true or false; it has the attribute and it's value is 'x'.  

I noticed that at the root in DirectoryView that @draggable is not created and therefore, for root I was thinking that it will not have the attribute at all.  

I didn't want the test to error out if the attribute doesn't exist.  

Of course, I'm brand new and don't know what I'm doing. ;)